conf/script/prov-m0-reset

```diff
@@ -485,6 +485,8 @@ mkfs() {
     done
 
     log "${FUNCNAME[0]}: Starting Hax.."
+    run_cmd $node1 "sudo mkdir -p /var/motr1/hax"
```

Comment:
when and who calls prov-m0-reset?
In case of data recovery, one node may be down. So ssh to other node might fail. Will that cause any problem? Or this script is run only when both the nodes are up.

Reply:
@ajaysrivas  - This script is called by Data Recovery in both the cases. The script prov-m0-reset is passed parameter --single-node if the other node is down so that prov-m0-reset can avoid calling ssh to the other node.